hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/utils/HddsServerUtil.java

```diff
@@ -109,6 +109,9 @@ public final class HddsServerUtil {
   private HddsServerUtil() {
   }
 
+  public static final String OZONE_RATIS_SNAPSHOT_COMPLETE_FLAG_NAME =
+      "OzoneRatisSnapshotCompleteFlag";
```

Comment:
nit: Since this is an actual file name, we could use all caps with hyphen or underline (just in case ):

```suggestion
  public static final String OZONE_RATIS_SNAPSHOT_COMPLETE_FLAG_NAME =
      "OZONE_RATIS_SNAPSHOT_COMPLETE";
```